src/main/java/net/minecraftforge/common/extensions/IForgeHolderSet.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.common.extensions;
+
+import net.minecraft.core.HolderSet;
+
+public interface IForgeHolderSet<T>
+{
+    private HolderSet<T> self()
```

Comment:
Self is only needed if the extension interface's default implementation itself needs to cast itself.